### ISSUE TITLE
Have a query page with search results

### ISSUE THREAD

**Issue by bedroesb** [elixir-europe/rdmkit#1017]
Would it be nice to have a query page we can link to with RDMkit results?

For example: https://rdmkit.elixir-europe.org/search?q=test

Gives all results for test. 

Feedback is welcome @elixir-europe/rdmkit-editors

**Comment by Nazeeefa** [comment]
Yes, only if query page displays exact search highlighted not results for each term e.g. if I search data brokering, I'd like to see a list of headings/text where both words are mentioned (and not results for the word 'data'). 

The example page says 'Page Not Found', was there supposed to be a different link?

**Comment by bedroesb** [comment]
oh no, this was just an example of the url, nothing real. What you are describing is more difficult to "fix" since I would use the same dependency as the current search for the results. SO a first iteration would be exactly like how the search bar works but than spread out on a page (next to the dropdown that now appears). Basically similar as the search on https://knowledgebase.nfdi4chem.de/knowledge_base/docs/intro/ with "all results in the dropdown"

**Comment by Nazeeefa** [comment]
Sounds good!

**Comment by martin-nc** [comment]
A search results page? Sounds okay, I just wonder what the use case is. We could send people a link like https://rdmkit.elixir-europe.org/search?q=data+brokering to show people everything on the site about data brokering, but wouldn't we just want to send them to the data brokering page instead? Are there going to be many cases where we don't have a page close to what people want?
